### PR TITLE
Business central Import settings Save

### PR DESCRIPTION
**Pull request by DhaaraniCIT**
## Summary by CodeRabbit

- **New Features**
  - Streamlined the import settings process for better user experience.
  - Enhanced tracking capabilities for onboarding steps with additional parameters.

- **Improvements**
  - Updated naming conventions for clarity in import settings services.
  - Refined form handling and validation in the import settings component.

- **Bug Fixes**
  - Fixed issues with variable declarations to ensure type safety and consistency.

- **Documentation**
  - Updated support article links to provide users with the most current information.

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
@coderabbitai review

File: src/app/integrations/business-central/business-central-shared/business-central-import-settings/business-central-import-settings.component.ts [hunk 1]
```diff
@@ -150,10 +153,45 @@ export class BusinessCentralImportSettingsComponent implements OnInit {
     this.importSettingWatcher();
   }
 
-  save() {
-    // Todo
+  constructPayloadAndSave() {
```
Comment (ashwin1111) [inline src/app/integrations/business-central/business-central-shared/business-central-import-settings/business-central-import-settings.component.ts:156]: private

**Review by ashwin1111 — APPROVED** [review]

File: src/app/core/services/common/helper.service.ts [hunk 2]
```diff
@@ -80,6 +80,11 @@ export class HelperService {
     return exportType ? new SnakeCaseToSpaceCasePipe().transform(new TitleCasePipe().transform(exportType)): 'expense';
   }
 
+  getTitleCaseValue(value: string): string {
```
Comment (ashwin1111) [inline src/app/core/services/common/helper.service.ts:83]: not needed

File: src/app/shared/components/configuration/configuration-import-field/configuration-import-field.component.html [hunk 3]
```diff
@@ -23,12 +23,30 @@
...
+    <div formArrayName="expenseFields" [ngClass]="[isDestinationFixedImport ? '' : 'tw-pl-48-px tw-mt-18px']">
+        <div *ngFor="let expenseField of expenseFieldsGetter.controls; let i = index" [formGroupName]="i" [ngClass]="{'tw-rounded-lg tw-border-separator tw-border tw-mt-24-px tw-pb-24-px': isDestinationFixedImport}">
+            <div *ngIf="isDestinationFixedImport">
+                <div class="tw-p-24-px tw-flex tw-justify-between tw-items-center">
+                    <div class="tw-w-3/4 tw-flex !tw-items-start">
+                      <div class="tw-pr-16-px tw-mt-[-10px]">
+                        <svg-icon-sprite src="import-setting" width="20px" height="20px" class="tw-text-mandatory-field-color"></svg-icon-sprite>
+                      </div>
+                      <div>
+                        <h4 class="tw-text-slightly-normal-text-color tw-text-16-px !tw-font-500 ">Import {{ helper.getTitleCaseValue(expenseField.value.destination_field) }} from {{ appName }}</h4>
```
Comment (ashwin1111) [inline src/app/shared/components/configuration/configuration-import-field/configuration-import-field.component.html:35]: `{{ expenseField.value.destination_field | snakeCaseToSpaceCase | titleCase }}`

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**